msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -24,7 +24,17 @@ The following schema sample shows all extensibility options.
     "name": "Publisher Name",
     "websiteUrl": "https://website.com/",
     "privacyUrl": "https://website.com/privacy",
-    "termsOfUseUrl": "https://website.com/app-tos"
+    "termsOfUseUrl": "https://website.com/app-tos",
+    "mpnId": "1234567890"
+  },
+  "localizationInfo": {
+    "defaultLanguageTag": "es-es",
```

Comment:
Its kind of weird that this example has default language set to es-es but all the default strings are english.

Reply:
fixed this one too that I forgot